package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "rudder-analytics",
-  "version": "1.2.8",
+  "version": "2.0.0",
```

Comment:
I think you should not change this before it is meged. And also, if it have to change a major version it means that that PR have breaking changes. So on the description of the PR you should mark the breaking change option

Reply:
@rivajunior Although these are not breaking changes, deviating from the general convention, we've changed the major version to indicate the big shift in our SDK functionality. Anyway, thanks for pointing it out and we hope you understand.